src/AlgAssAbsOrd/Conjugacy/Lifting.jl

```diff
@@ -230,8 +235,9 @@ function Base.divrem(n::T, m::T) where T <: Union{nmod,Nemo.fmpz_mod}
       end
     end
   end
-  qq =  R(crt([x[2] for x = q], [x[1] for x = q])::fmpz)::T
+  qq = R(crt([x[2] for x = q], [x[1] for x = q])::fmpz)::T
   rr = n-qq*m
+  @show rr
```

Comment:
you want to show this?

Reply:
You should use the suggestion feature more.